packages/google_sign_in/google_sign_in_ios/pubspec.yaml

```diff
@@ -2,7 +2,7 @@ name: google_sign_in_ios
 description: iOS implementation of the google_sign_in plugin.
 repository: https://github.com/flutter/packages/tree/main/packages/google_sign_in/google_sign_in_ios
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+google_sign_in%22
-version: 5.7.5
+version: 5.7.6
```

Comment:
Hm, do you think this should be a minor bump instead @stuartmorgan?

Reply:
Since this one *should* be a no-op for clients, bugfix seems fine. I don't feel strongly between the two here.